src/python/WMCore/Services/WMStats/WMStatsWriter.py

```diff
@@ -43,7 +43,7 @@ def convertToServiceCouchDoc(wqInfo, wqURL):
     wqDoc = {}
     wqDoc['_id'] = wqURL
     wqDoc['agent_url'] = wqURL
-    wqDoc['agent_team'] = ""
```

Comment:
if we remove agent_team then we also have to update the javascript code. Either one is fine with me